packages/edge-login-ui-rn/src/common/locales/strings/de.json

```diff
@@ -56,7 +56,6 @@
   "terms_two": "Ich verstehe, dass wenn ich dieses Gerät verliere, oder die App deinstalliere, mein digitales Vermögen nur noch mit Benutzernamen und Passwort wiederhergestellt werden können",
   "terms_three": "Ich verstehe, dass wenn ich meinen Benutzernamen und mein Passwort verliere, Edge mein Benutzerkonto nicht wiederherstellen kann, sollte vorher nicht die Passwort-Wiederherstellung aktiviert worden sein",
   "terms_four": "Ich verstehe, dass ich für die Sicherstellung meiner Passwörter, privaten Schlüsselpaare, PIN und alle anderen Codes für den Zugriff auf die Software verantwortlich bin. %s ist nicht verantwortlich, wenn meine Informationen kompromittiert werden oder von Dritten aufgerufen werden, bei denen das Geld verloren geht",
-  "step_one": "Schritt 1/3",
```

Comment:
When editing strings, please don't touch the translations. We have automated tools to manage this.